### PR TITLE
Add cilium-enterprise-mixin

### PR DESCRIPTION
**Pull request by Dasomeone**
Adds a new mixin for Cilium Enterprise.

Dashboards are included, but alerts are currently disabled and being iterated on.

### Dashboards
- Cilium / Overview
- Cilium / Operator
- Cilium / Agent Overview
- Cilium / Components / Actionable
- Cilium / Components / Agent
- Cilium / Components / API
- Cilium / Components / BPF
- Cilium / Components / Conntrack
- Cilium / Components / Datapath
- Cilium / Components / External HA FQDN Proxy
- Cilium / Components / FQDN Proxy
- Cilium / Components / Identities
- Cilium / Components / Kubernetes
- Cilium / Components / L3 Policy
- Cilium / Components / L7 Proxy
- Cilium / Components / Network
- Cilium / Components / Nodes
- Cilium / Components /Policy
- Cilium / Components / Resource Utilization
- Hubble / Overview
- Hubble / Timescape
cilium_agent_overview_1
hubble_timescape_1
cilium_overview_1
hubble_overview_1
cilium_operator_overview_1

### CODE REVIEW

**Review by gloriasc — APPROVED** [review]
Looks super good @Dasomeone ! Thanks for walking me through how to review @gaantunes

**Review by gaantunes — APPROVED** [review]
LGTM, nicely done!

File: cilium-enterprise-mixin/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+```bash
+$ go get github.com/monitoring-mixins/mixtool/cmd/mixtool
+$ go get github.com/google/go-jsonnet/cmd/jsonnetfmt
+```
+
+You can then build the Prometheus rules files `alerts.yaml` and
+`rules.yaml` and a directory `dashboard_out` with the JSON dashboard files
+for Grafana:
+```bash
+$ make build
```
Comment (stevo-f3) [inline cilium-enterprise-mixin/README.md:93]: @Dasomeone thanks for sharing ❤️ 

Just an observation, Makefile is missing, unlike for other mixins; was it intentional to leave it out?

File: cilium-enterprise-mixin/dashboards/cilium-agent-overview.json [hunk 2]
```diff
@@ -0,0 +1,1688 @@
...
+        }
+      ],
+      "title": "Top Total BPF Maps Memory Usage",
+      "type": "timeseries"
+    }
+  ],
+  "refresh": "30s",
+  "schemaVersion": 36,
+  "style": "dark",
+  "tags": [],
```
Comment (stevo-f3) [inline cilium-enterprise-mixin/dashboards/cilium-agent-overview.json:1522]: @Dasomeone tags are being referenced in links in the dashboards, but tags aren't being set for the dashboards; am I missing something?

File: cilium-enterprise-mixin/README.md (hunk 1 above)
Comment (Dasomeone) [reply cilium-enterprise-mixin/README.md:93]: Sorry for the late reply, been off on vacation, but yes indeed the makefile is missing! Simply forgot to include it :)

Will push a follow-up PR for this in a bit.

File: cilium-enterprise-mixin/dashboards/cilium-agent-overview.json (hunk 2 above)
Comment (Dasomeone) [reply cilium-enterprise-mixin/dashboards/cilium-agent-overview.json:1522]: No you're absolutely right, in the use-case I have for the mixin, I apply the tags elsewhere and overlooked that it wouldn't be included with the base mixin.
Will add them back in :)

**Pull request merged**